src/configurations/zhimi_humidifier_ca4_xiaomi_miio_airpurifier.js

```diff
@@ -0,0 +1,133 @@
+import ICON from '../const';
+
+// for integration https://github.com/syssi/xiaomi_airpurifier
+// because xiaomi_miio from home assistant not support water level
+const ZHIMI_HUMIDIFIER_CA4_XIAOMI_MIIO_AIRPURIFIER.js = () => ({
```

Comment:
```suggestion
const ZHIMI_HUMIDIFIER_CA4_XIAOMI_MIIO_AIRPURIFIER = () => ({
```

@ravikwow I think you want to remove this suffix